src/algebra/char_p/basic.lean

```diff
@@ -177,6 +177,18 @@ theorem sub_pow_char_pow [comm_ring R] {p : ℕ} [fact p.prime]
   (x - y) ^ (p ^ n) = x ^ (p ^ n) - y ^ (p ^ n) :=
 sub_pow_char_pow_of_commute _ _ _ (commute.all _ _)
 
+open_locale big_operators
+
+lemma sum_pow_char {ι : Type*} [comm_semiring R] {p : ℕ} [fact p.prime] [char_p R p]
+  (s : finset ι) (f : ι → R) :
+  (∑ i in s, f i) ^ p = ∑ i in s, f i ^ p :=
```

Comment:
If you move this below the definition of `frobenius`, then you can use `(frobenius R).map_sum _` as a proof.

Reply:
Nice find, thanks!